benches/rate_calculation.rs

```diff
@@ -139,8 +139,8 @@ fn criterion_benchmark(c: &mut Criterion) {
     // Define timestep
     world.insert(Timestep { delta: 1.0e-6 });
 
-    let vel_dist = Normal::new(0.0, 0.22);
-    let pos_dist = Normal::new(0.0, 1.2e-4);
+    let vel_dist = Normal::new(0.0, 0.22).unwrap();
```

Comment:
Here, a compile error.

Reply:
Thanks! the benches are a bit unfinished for now, I will create a new issue to add more benchmarks to the bench suite.